packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -136,39 +135,35 @@ class Auth {
             fetchOptions: config.fetchOptions
         })
 
-        if (this.get('site_id') && this.get('site_id') !== config.siteId) {
-            // if site is switched, remove all existing auth data in storage
-            // and the next auth.ready() call with restart the auth flow
-            this.clearStorage()
-            this.pendingToken = undefined
-        }
-
-        if (!this.get('site_id')) {
-            this.set('site_id', config.siteId, {
-                expires: this.REFRESH_TOKEN_EXPIRATION_DAYS
-            })
-        }
+        // NOTE: Not too sure how I feel about this, seems a little confusing. Should
```

Comment:
Thoughts on this would be appreciated. It isn't horrible, but I think it could be cleaner.